app/src/androidTest/java/com/mapbox/services/android/navigation/ui/v5/NavigationMapRouteTest.java

```diff
@@ -1,200 +0,0 @@
-package com.mapbox.services.android.navigation.ui.v5;
-
-import android.support.test.espresso.Espresso;
-import android.support.test.espresso.IdlingResourceTimeoutException;
-import android.support.test.espresso.UiController;
-import android.support.test.rule.ActivityTestRule;
-import android.support.test.runner.AndroidJUnit4;
-
-import com.mapbox.mapboxsdk.constants.Style;
-import com.mapbox.mapboxsdk.maps.MapboxMap;
-import com.mapbox.mapboxsdk.style.layers.Property;
-import com.mapbox.services.android.navigation.testapp.activity.navigationui.NavigationMapRouteActivity;
-import com.mapbox.services.android.navigation.ui.v5.route.NavigationMapRoute;
-import com.mapbox.services.android.utils.OnMapReadyIdlingResource;
-
-import org.junit.After;
-import org.junit.Before;
-import org.junit.Rule;
-import org.junit.Test;
-import org.junit.runner.RunWith;
-
-import timber.log.Timber;
-
-import static android.support.test.espresso.Espresso.onView;
-import static android.support.test.espresso.assertion.ViewAssertions.matches;
-import static android.support.test.espresso.matcher.ViewMatchers.isDisplayed;
-import static android.support.test.espresso.matcher.ViewMatchers.withId;
-import static junit.framework.Assert.assertTrue;
-
-@RunWith(AndroidJUnit4.class)
```

Comment:
Maybe you could just remove this line and it won't run? So you don't have to remove the entire file. (I'm not sure if that'll work- I would double check)